components/selection/selection-summary.js

```diff
@@ -35,25 +36,54 @@ class Summary extends LocalizeCoreElement(SelectionObserverMixin(LitElement)) {
 	}
 
 	render() {
+		if (!this._summary) {
+			return nothing;
+		}
+		return html`
+			<p class="d2l-body-compact">${this._summary}</p>
+		`;
+	}
+
+	updated(changedProperties) {
```

Comment:
It looks like the purpose of this is to compute `includePlus`, and ultimately the summary text that will get rendered on a subsequent pass of the render life cycle. Using `updated` followed by updating a reactive property (`_summary` in this case) is the classic way to handle this in Lit 1. 

Lit 2 provides a new life cycle callback that enables us to avoid the second iteration of the event loop, [willupdate](https://lit.dev/docs/components/lifecycle/#willupdate). It runs before `render`, and enables us to compute (non-reactive) properties that can be used by `render`, thereby avoiding the second update. I don't think we've used `willupdate` in core yet... but I think it would be an improvement here. It's something I think we could use in a few other places in core as we make revisions.

Reply:
Interesting, didn't know that! That is handy. Seems to work as expected so I'll push the changes. 🎉